.github/workflows/docs.yml

```diff
@@ -64,11 +64,11 @@ jobs:
             echo "::warning title=Invalid file permissions automatically fixed::$line"
           done
       - name: Setup Pages
-        uses: actions/configure-pages@v3
-      - name: Upload artifact
-        uses: actions/upload-pages-artifact@v2
-        with:
+        uses: actions/configure-pages@v5
+              - name: Upload artifact
+        uses: actions/upload-pages-artifact@v3
+                with:
```

Comment:
```suggestion
     - name: Upload artifact
        uses: actions/upload-pages-artifact@v3
        with:
```

Reply:
Unless I'm missing something, I didn't see why we needed to change the indentation.